process.ts

```diff
@@ -495,23 +494,35 @@ async function taskLoadModule(
   }
 }
 
+let moduleIndex;
 async function taskAlgolia(
   id: number,
   { module, version }: AlgoliaTask,
 ) {
   console.log(
-    `[${id}]: %cUploading%c %c"${version.name}@${version.version}"%c doc nodes to algolia...`,
+    `[${id}]: %cUploading%c %c"${version.name}@${version.version}"%c to algolia...`,
     "color:green",
     "color:none",
     "color:cyan",
     "color:none",
   );
-  const requests: MultipleBatchRequest[] = [];
-  requests.push(moduleToRequest(module));
-  await loadDocNodes(requests, module, version);
-  await upload(requests);
+
+  moduleIndex ??= (await getDenoLandApp()).initIndex(indexes.MODULE_INDEX);
+  await moduleIndex.saveObject({
```

Comment:
Why `saveObject` instead of update?

Reply:
there is no updateObject, instead saveObject overwrites if there is an existing one